### ISSUE TITLE
Release schedule for 0.17.0

### ISSUE THREAD

**Issue by laanwj** [bitcoin/bitcoin#12624]
Here is a proposed release schedule for 0.17.0, the next major release of Bitcoin Core. Like for previous major releases I've aimed for a release 6 months after the last (#11449).
```
2018-07-02
-----------
- Open Transifex translations for 0.17
- Soft translation string freeze (no large or unnecessary string changes until release)
- Finalize and close translations for 0.15

2018-07-23
-----------
- Feature freeze (bug fixes only until release)
- Translation string freeze (no more source language changes until release)

2018-08-13
-----------
- Split off `0.17` branch from `master`
- Start RC cycle, tag and release `0.17.0rc1`
- Start merging for 0.18 on master branch

2018-09-08
-----------
- Release 0.17.0 final (aim)
```
If any specific dates or timeframes are a problem for you, let me know.

**Comment by AlexGoldovsky** [comment]
@laanwj Excuse me, but, What exactly is a good first issue about this?
I'm looking for an issue to start with, but I don't get what should be done here.

**Comment by laanwj** [comment]
@AlexGoldovsky If you plan on submitting any PRs, it's good to know about the upcoming release schedule.

**Comment by thijstriemstra** [comment]
> Excuse me, but, What exactly is a good first issue about this?

Was thinking the same; sounded like being a release manager as first issue/task would be good.

**Comment by fanquake** [comment]
@thijstriemstra The "good first issue" label was originally added to release milestones as a bit of "tongue in cheek", however as @laanwj has pointed out, if new contributors are filtering for issues using that label, they will now also see the current release schedule.

**Comment by laanwj** [comment]
Right - it's a good first issue *to read*.

**Comment by flack** [comment]
This discussion about "good first issue" happens on every single one of these release schedule tickets. Maybe it's time to admit defeat and remove it? If I'm a new contributor looking at "good first issues", I'm looking for something to work on. If I want to know when the next release happens, I go to https://github.com/bitcoin/bitcoin/milestones

Thinking a bit further, github milestones support both a due date and a description, so why not use those? The release schedule ticket is not really actionable anyways, so it might be better to simply use the facilities provided by the platform

**Comment by flack** [comment]
Sorry, I didn't mean to sound bossy or anything. But seeing how good first issue is supposed to be for new contributors, I thought maybe an outsider's perspective might be helpful. Obviously, do what you think is best, I just find it silly that literally on each of these release tickets it's the same discussion

**Comment by flack** [comment]
One small (and hopefully uncontroversial) change that could be made would be to link this ticket from the milestone's description (i.e. just put in something like "see #12624 for proposed schedule"). It would improve discoverability and you could still keep the tag if you really want to :-)

**Comment by mmortal03** [comment]
Also, shouldn't this one be closed now that 0.17.0 was released?
